src/OnnxSharp.Test/OnnxSharp.Test.csproj

```diff
@@ -7,7 +7,7 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <TargetFrameworks>net461;net5.0</TargetFrameworks>
+    <TargetFrameworks>net462;net5.0</TargetFrameworks>
```

Comment:
why change target?

Reply:
It was a warning from Visual Studio. If I recall correctly, `System.Text.Json` requires at least .NET Framework 4.6.2.

Reply:
👍